### PR TITLE
Enable Renovate to manage node versions

### LINKED ISSUE #26: Maintain node versions
Ensure that our configuration will manage Node versions for us in the following files:

* `.circleci/config.yml`
* `.nvmrc`
* `package.json`

### PR DESCRIPTION
**Pull request by sjparkinson**
We would like Renovate to open pull requests on our repositories when there are new versions of node available.

There are several files in our repository that will need changing...

* `.nvmrc`
* `.circleci/config.yml`
* `package.json`

We have already pinned the version of node used in all of our apps. With Renovate opening pull requests to upgrade node, we can release upgrades in isolation to code changes.

Resolves #26.

### CODE REVIEW

**Comment by sjparkinson** [comment]
See https://github.com/financial-times-sandbox/testing-123/pull/14 for an example pull request that Renovate will open on our repositories.

**Comment by sjparkinson** [comment]
We're not quite there, as you can see in the example repository we're not grouping all node upgrades into a single pull request yet.

I'll open an issue on the Renovate configuration help repository to figure this out.

**Comment by sjparkinson** [comment]
Should we be merging this before we have sorted out supporting Node.js 8.15.0 and beyond?

**Comment by simonplend** [comment]
@sjparkinson I don't think so, otherwise I think we'll end up in the same situation we were in before we pinned to node 8.13.0.

File: renovate.json [hunk 1]
```diff
@@ -20,6 +21,9 @@
   "labels": [
     "dependencies"
   ],
+  "supportPolicy": [
+    "lts_active"
```
Comment (sjparkinson) [inline renovate.json]: See https://renovatebot.com/docs/node/#configuring-support-policy for more information on this one.

The tl;dr is this specifies we are only interested in upgrades to LTS versions.

Even more information at https://github.com/nodejs/Release#release-schedule.

Comment (sjparkinson) [reply renovate.json]: I've updated this to the `lts` policy, as we're happy under the maintenance period too.

**Comment by sjparkinson** [comment]
Closing this to open as a draft pull request until we're ready to release this feature.

**Pull request closed without merging**